src/localplayer.cpp

```diff
@@ -151,9 +152,345 @@ static bool detectSneakLadder(Map *map, INodeDefManager *nodemgr, v3s16 pos)
 
 #undef GETNODE
 
+// For classic sneak behaviour. Disabled by default and possibly temporary
+void LocalPlayer::classic_move(f32 dtime, Environment *env, f32 pos_max_d,
+		std::vector<CollisionInfo> *collision_info)
+{
+	Map *map = &env->getMap();
+	INodeDefManager *nodemgr = m_client->ndef();
+
+	v3f position = getPosition();
```

Comment:
Can't be const... it's modified by more than one line below (see e.g. line 179, 273 etc)